desc/objectives/_bootstrap.py

```diff
@@ -251,25 +251,21 @@ def build(self, use_jit=True, verbose=1):
 
         super().build(use_jit=use_jit, verbose=verbose)
 
-    def compute(self, params, constants=None):
+    def compute(self, params):
         """Compute the bootstrap current self-consistency objective.
 
         Parameters
         ----------
         params : dict
             Dictionary of equilibrium degrees of freedom, eg Equilibrium.params_dict
-        constants : dict
-            Dictionary of constant data, eg transforms, profiles etc. Defaults to
-            self.constants
 
         Returns
         -------
         obj : ndarray
             Bootstrap current self-consistency residual on each rho grid point.
 
         """
-        if constants is None:
-            constants = self.constants
+        constants = self.constants
```

Comment:
I'd recommend leaving it how it was, but throw a deprecation warning `if constants is not None`